chapters/naming-things.qmd

```diff
@@ -0,0 +1,733 @@
+
+## Biology
+
+```{zsh}
+01_marshal-data.r
+02_pre-dea-filtering.r
+03_dea-with-limma-voom.r
+04_explore-dea-results.r
+90_limma-model-term-name-fiasco.r
+helper01_load-counts.r
+helper02_load-exp-des.r
+helper03_load-focus-statinf.r
+helper04_extract-and-tidy.r
+```
+
+:::
+
+#### Use the ISO 8601 standard for dates
+
+`YYYY-MM-DD`
+
+::: panel-tabset
+
+## Psychology
+
+```{zsh filename="Output"}
+#| code-copy: false
+2021-02-18_NEOFFI_numerical-stroop_bsc-thesis_01.csv
+2021-02-18_NEOFFI_numerical-stroop_bsc-thesis_02.csv
+2021-02-19_NEOFFI_numerical-stroop_bsc-thesis_03.csv
+2021-06-04_NEOFFI_emotional-stroop_msc-thesis_001.csv
+2021-06-05_NEOFFI_emotional-stroop_msc-thesis_002.csv
+2021-06-07_NEOFFI_emotional-stroop_msc-thesis_003.csv
+2021-06-11_NEOFFI_emotional-stroop_msc-thesis_004.csv
+```
+
+## Biology
+
+```{zsh filename="Output"}
+#| code-copy: false
+2013-06-26_BRAFWTNEGASSAY_Plasmid-Cellline-100-1MutantFraction_H01.csv
+2013-06-26_BRAFWTNEGASSAY_Plasmid-Cellline-100-1MutantFraction_H02.csv
+2013-06-26_BRAFWTNEGASSAY_Plasmid-Cellline-100-1MutantFraction_H03.csv
+2014-02-17_BRAFWTNEGASSAY_FFPEDNA-CRC-1-41_A01.csv
+2014-02-17_BRAFWTNEGASSAY_FFPEDNA-CRC-1-41_A02.csv
+2014-02-17_BRAFWTNEGASSAY_FFPEDNA-CRC-1-41_A03.csv
+2014-02-17_BRAFWTNEGASSAY_FFPEDNA-CRC-1-41_A04.csv
+```
+
+:::
+
+```{r}
+#| results: asis
+#| eval: true
+#| echo: false
+cat(make_figure("xkcd-iso-8601"))
+```
+
+#### Consider the output:
+
+Files that are later turned into HTML should be exclusively delimited by hyphens (`-`).
+
+### Left-pad numbers with zeros
+
+Left-padding of numbers refers to the practice of adding zeros to the left of a number to make it reach a certain length.
+This can be useful, for example, when dealing with numerical data that needs to be formatted in a specific way for presentation or computational purposes.
+By left-padding numbers, you can ensure that they all have the same length and are easier to compare or work with in a numerical context.
+
+::: panel-tabset
+
+## Psychology
+
+```{zsh}
+01_DSM-5-data.csv
+02_pre-analysis-filtering.r
+03_test-hypotheses.r
+04_exploratory-analyses.r
+90_DSM-V-codebook.csv
+helper01_load-packages.r
+helper02_load-codebook.r
+helper03_test-assumptions.r
+helper04_load-figure-template.r
+```
+
+## Biology
+
+```{zsh filename="Output"}
+#| code-copy: false
+01_marshal-data.r
+02_pre-dea-filtering.r
+03_dea-with-limma-voom.r
+04_explore-dea-results.r
+90_limma-model-term-name-fiasco.r
+helper01_load-counts.r
+helper02_load-exp-des.r
+helper03_load-focus-statinf.r
+helper04_extract-and-tidy.r
+```
+
+:::
+
+If you don't left pad numbers, you get this:
+
+```{zsh filename="Output"}
+#| code-copy: false
+10_final-figs-for-publication.R
+1_data-cleaning.R
+2_fit-model.R
+```
+
+## Summary
+
+### Machine readable
+
+1. File names are regular expression and globbing friendly
+1. File names allow to recover meta data
+
+### Human readable
+
+Add scss code to disable the code-copy button on selected code chunks
```

Comment:
Is this sentence at the correct position?

Reply:
No, removed this.